etcdadm/etcdadm

```diff
@@ -332,9 +332,9 @@ _docker_awscli_command() {
   dir=$(dirname "$(config_state_dir)")
   echo docker
   echo run
+  echo --rm
   printf "%s\n" "-e"
   echo "AWS_DEFAULT_REGION=$aws_region"
-  echo "--rm"
```

Comment:
it *should* change nothing but it actually solved the issue in my case, i'm 100% sure.  I will need some time for reproduction to explain that :) The way how the command line gets constructed here is a bit confusing